### DISCUSSION TITLE
A question from someone almost new

### DISCUSSION THREAD

**Discussion by 7HRAMZI** [lllyasviel/Fooocus#2635] [Q&A] [answered]
Question from someone almost new...

Hi, I have a question: Should I use Nvidia 551.86 driver update that is active on my device or should I use Nvidia 531 driver update (not tested on my device), which update should I use "Nvidia 531" is it 531.41 or 531.61 ? Or continue using Nvidia update 551.86 ?

Who is the best Nvidia driver update for Fooocus ?

BTW MY GPU V-RAM IS 4GB.

Additionally, does the wizard play a role in Fooocus ?
If yes, how many cores and threads should be used (recommended requirements) ?

Additionally, What is the best and fastest system for Fooocus ?

BTW MY OS IS WINDOWS 10 PRO

Sorry if it's a stupid or repetitive question

**Comment by mashb1t** [comment, marked as answer]
Please find the minimum system specs in the readme.
System specs also see https://github.com/lllyasviel/Fooocus/discussions/2594#discussioncomment-8870721, best is 4090, A100 or H100.

**Reply by mashb1t** [reply]
using the latest drivers is almost always best. Your VRAM seems quite small, don't expect too mich performance as much swap will be used and GPU is your bottleneck.

**Reply by 7HRAMZI** [reply]
Thank you very much
